compiler-core/templates/gleam@@compile.erl

```diff
@@ -7,14 +7,31 @@
 
 main(Args) ->
     #arguments{out = Out, lib = Lib, modules = Modules} = parse(Args),
+    IsElixirModule = fun(Module) ->
+        filename:extension(Module) =:= ".ex"
+    end,
+    {ElixirModules, ErlangModules} = lists:partition(IsElixirModule, Modules),
     ok = configure_logging(),
     ok = add_lib_to_erlang_path(Lib),
     ok = filelib:ensure_dir([Out, $/]),
+    {ErlangOk, ErlangBeams} = compile_erlang(ErlangModules, Out),
+    {ElixirOk, ElixirBeams} = compile_elixir(ElixirModules, Out),
```

Comment:
I think we don't want to run the Elixir build if the Erlang build has finished, it likely depends on the Erlang modules being present.